lib/database_cleaner/null_strategy.rb

```diff
@@ -1,18 +1,18 @@
 module DatabaseCleaner
   class NullStrategy
-    def self.start
+    def start
```

Comment:
this is an API change
```
DatabaseCleaner.strategy = DatabaseCleaner::NullStrategy
```
woun't work anymore, right?

Reply:
Ah, I assumed that the null strategy was only used internally. Upon reflection, that is obviously not a safe assumption! This change was tangential to the main thrust of the PR, anyways, so I'll just kill that particular commit. Thanks for catching this!